mods/ts/maps/sunstroke/map.yaml

```diff
@@ -29,14 +29,14 @@ Players:
 		Enemies: Multi0, Multi1
 	PlayerReference@CreepsNodBase:
 		Name: CreepsNodBase
-		Color: FE1100
```

Comment:
These aren't changing anything in practice, so can you drop them from the commit?